src/cockpit/packages.py

```diff
@@ -81,6 +82,11 @@ def get_libexecdir() -> str:
 
 def patch_libexecdir(obj):
     if isinstance(obj, str):
+        if '${libexecdir}/cockpit-askpass' in obj:
+            # extra-special case: we handle this internally
+            abs_askpass = shutil.which('cockpit-askpass')
```

Comment:
I'm rather surprised that this doesn't raise an exception when not found, but it doesn't.

Reply:
I was equally surprised when my tests were suddenly failing with a message about how .replace() doesn't like `None` as its second argument :)